cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -5,7 +5,7 @@ namespace: default
 labels:
   k8s-app: skydns
 desiredState:
-  replicas: {DNS_REPLICAS}
+  replicas: {{ pillar['dns_replicas'] }}
```

Comment:
Will check and update. It became more convoluted on our end, certainly - not sure about the end-users side, though, hopefully.